src/specBuilder/signal/signalSpecBuilder.ts

```diff
@@ -79,9 +85,27 @@ export const getGenericSignal = (name: string, value: unknown = null): Signal =>
  * @param signals
  * @param markName
  * @param datumOrder how deep the datum is nested (i.e. 1 becomes datum.rscMarkId, 2 becomes datum.datum.rscMarkId, etc.)
+ * @param animations
+ * @param animateFromZero
+ * @param needsDisable
  */
-export const addHighlightedItemSignalEvents = (signals: Signal[], markName: string, datumOrder = 1) => {
+export const addHighlightedItemSignalEvents = ({
+	signals,
+	markName,
+	datumOrder = 1,
+	animations = false,
+	animateFromZero = false,
+	needsDisable = false,
+}: {
+	signals: Signal[];
+	markName: string;
+	datumOrder?: number;
+	animations?: boolean;
+	animateFromZero?: boolean;
+	needsDisable?: boolean;
```

Comment:
I think falsy boolean variables can end up being more difficult to understand intuitively. Would this make more sense as the opposite? `isEnabled`